changelogs/unreleased/fiber-basic-api.md

```diff
@@ -0,0 +1,7 @@
+## feature/fiber
+
+* Exported `fiber_set_name_n`, `fiber_get_name`, `fiber_get_id`,
+  `fiber_get_csw` and `fiber_find` into the public C API
+  and usable via FFI as well.
+* Make `fiber_set_joinable`, `fiber_set_ctx` and `fiber_get_ctx`
+  treat the NULL argument as the current fiber.
```

Comment:
The changelog has to be updated with the new export names since the previous review.

Reply:
done